mergJSON/mergJSONLibrary.livecodescript

```diff
@@ -0,0 +1,94 @@
+﻿script "ws.goulding.script-library.mergjson"
+/**
+
+Name: mergJSON Support Library
+
+Summary:
+Support functions for the mergJSON external
```

Comment:
Could you please expand this top-level documentation comment with an explanation of why you need this library in addition to the external itself?  It would also be great to add at least one `Example:` for each command/function in the library.